acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -201,4 +168,46 @@ public void tearDownAcceptanceTestBase() {
       LOG.info("Don't know how to report memory for OS {}", os);
     }
   }
+
+  @Rule
```

Comment:
Remind me again why this rule needs to be in the `AcceptanceTestBase` and not it's own class?

Reply:
Its purpose is that of @Before and @After, so I thought here it makes conceptual sense. But I guess it could go to its own class. Let me know whether to separate it then.

Reply:
By the way, the _other_ @Rule was unused and just a vestige of a previous implementation. I'll remove it.

Reply:
(... or should that be another PR?)